applications/openshift/etcd/etcd_peer_key_file/oval/shared.xml

```diff
@@ -30,4 +31,10 @@
     <ind:subexpression datatype="string" operation="pattern match">^/etc/etcd/peer.key$</ind:subexpression>
   </ind:textfilecontent54_state>
 
+{{%- if product == "ocp4" %}}
+  {{{ oval_ocp_service_runtime_config(command="etcd", option="--peer-key-file", value="/etc/ssl/etcd/system\:etcd-peer\:etcd.*\.key", option_id="peer_key_file") }}}
```

Comment:
XCCDF has:
``ETCD_PEER_KEY_FILE=/etc/ssl/etcd/system:etcd-peer:<i><etcd_dns_name</i>.key``

But OVAL has:
``value="/etc/ssl/etcd/system\:etcd-peer\:etcd.*\.key"``

Which is correct?

Reply:
Or is there a naming convention that etcd dns names will have ``etcd*``?

Reply:
@shawndwells I don't think that you can guarantee an FQDN is going to be the same at all.

Reply:
@redhatrises right, so shouldn't the oval be:

``value="/etc/ssl/etcd/system\:etcd-peer\:.*\.key"``

vs the current 
``value="/etc/ssl/etcd/system\:etcd-peer\:etcd.*\.key"``

Reply:
Good point. I see what you are saying now. Fixed.